common.sh

```diff
@@ -269,6 +271,40 @@ export EXTRA_WORKER_MEMORY=${EXTRA_WORKER_MEMORY:-${WORKER_MEMORY}}
 export EXTRA_WORKER_DISK=${EXTRA_WORKER_DISK:-${WORKER_DISK}}
 export EXTRA_WORKER_VCPU=${EXTRA_WORKER_VCPU:-${WORKER_VCPU}}
 
+if [[ ! -z ${E2E_TEST_SCENARIO} ]]; then
+  export IP_STACK=${E2E_TEST_SCENARIO##*_IP}
+
```

Comment:
minor optimization: scenario base name could be extracted by using 

```SCENARIO=${E2E_TEST_SCENARIO%%_*}```

and then it could be used to simplify the switch cases

Reply:
Done